### PR TITLE
better handle anonymous users

### PR DESCRIPTION
**Pull request by leggsimon**
## Todo
 - [x]  ~Add test for bailing early if user is anon~

### CODE REVIEW

**Comment by leggsimon** [comment]
@debugwand I wonder whether this should really be a major version release since we're changing how the API can return?

File: src/myft-client.js [hunk 1]
```diff
@@ -33,6 +33,12 @@ class MyFtClient {
 			return Promise.resolve();
 		}
 		this.initialised = true;
+
+		const anonymousUser = !(/FTSession=/.test(document.cookie))
```
Comment (debugwand) [inline src/myft-client.js]: semi-colon missing

**Comment by debugwand** [comment]
is it a breaking change? then it is a major release

**Comment by leggsimon** [comment]
Well I guess it _could_ be a breaking change since we are rejecting based on something that we weren't before...

**Comment by debugwand** [comment]
yes breaking change. release as a major and then you can roll out app by app and test for regressions

**Comment by leggsimon** [comment]
Ok, what do you think on this then

>If so I wonder whether we could look at improving the updateRelationship api that we discussed a few weeks ago.

**Comment by debugwand** [comment]
going to slack

**Pull request merged**